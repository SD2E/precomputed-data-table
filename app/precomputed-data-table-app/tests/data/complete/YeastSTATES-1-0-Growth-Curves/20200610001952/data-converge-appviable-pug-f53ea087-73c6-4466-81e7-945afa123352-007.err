/opt/conda/envs/sd2-data_converge/lib/python3.7/site-packages/FlowCytometryTools/core/docstring.py:56: MatplotlibDeprecationWarning: 
The dedent function was deprecated in Matplotlib 3.1 and will be removed in 3.3. Use inspect.cleandoc instead.
  doc = dedent(doc)
/opt/conda/envs/sd2-data_converge/lib/python3.7/site-packages/FlowCytometryTools/core/docstring.py:56: MatplotlibDeprecationWarning: 
The dedent function was deprecated in Matplotlib 3.1 and will be removed in 3.3. Use inspect.cleandoc instead.
  doc = dedent(doc)
