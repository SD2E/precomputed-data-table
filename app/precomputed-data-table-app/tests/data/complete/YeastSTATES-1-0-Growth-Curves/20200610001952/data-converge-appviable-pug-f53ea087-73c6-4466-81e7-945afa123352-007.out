container_exec reshg/data-converge-app:0.1.0 /opt/conda/envs/sd2-data_converge/bin/python /data_converge/src/data_converge/run_data_converge.py --experiment_ref YeastSTATES-1-0-Growth-Curves
/opt/apps/tacc-singularity/2.6.0/bin/singularity
singularity exec: reshg/data-converge-app:0.1.0 /opt/conda/envs/sd2-data_converge/bin/python /data_converge/src/data_converge/run_data_converge.py --experiment_ref YeastSTATES-1-0-Growth-Curves
Docker image path: index.docker.io/reshg/data-converge-app:0.1.0
Cache folder set to /work/06935/sgopal/singularity_cache/docker
Creating container runtime...
Exploding layer: sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b.tar.gz
Exploding layer: sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3.tar.gz
Exploding layer: sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b.tar.gz
Exploding layer: sha256:ff584b4ef2262162a09c97cba9b4f701610524b5c559367704cbeefccadb5f4d.tar.gz
Exploding layer: sha256:186061fcbfe91d394b631b20aad7253539d086c9d9ecc975aeac2e074604fbcc.tar.gz
Exploding layer: sha256:708a39bff4a5e539e6a9b738f3699d6c965cf878997a71dae7273faa5559e98e.tar.gz
Exploding layer: sha256:f46bd54e2d5e3e974a91e854f27a5e036e9226f0d134ad9382e2069a2f190758.tar.gz
Exploding layer: sha256:6f6f0be712244267781d8aa28432d83ffed1c2b125770b1e04714ac2b55b1fbf.tar.gz
Exploding layer: sha256:463fefc24c8869572b3ea4a19c8057e868b4d5d20bdf6155e2f833bade0b8f4a.tar.gz
Exploding layer: sha256:9318fe5d63a8c51c1c9666998e60a664d588ad836407893f0834c0ada4f30b1c.tar.gz
processing data sets...
fc raw:  no records []
fc etl:  no records []
plate reader:  ok. unique sample ids: 830 []
summarizing data sets...
hashing data sets...
MD5: a440e99a735b90aae3dec977fd7144e5
making experiment record...
making product record...
finished!
passing archive_system step
